### PR TITLE
Topic recognition - Solution for AKOA Laterality Classification

### PR DESCRIPTION
**Pull request by gmulhearn**
A solution for AKOA Knee Laterality Classification and driver script. please refer to README.md 📝 for usage/descriptions.

By student s4532094 - George Mulhearn

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Easy Difficulty
- Performance appears good

Other Feedback
- None, good job

**Comment by rexsw** [comment]
Code looks good for normal difficulty

Love the emojis 

For next time;

Tensorflow has an MSE function so there's no need to write one yourself, make sure you use your libraries fully

File: recognition/laterality_classification/driver.py [hunk 1]
```diff
@@ -0,0 +1,305 @@
+from recognition.laterality_classification.laterality_classifier import *
```
Comment (shakes76) [inline recognition/laterality_classification/driver.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Fix the conflict with the .gitignore file.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by gmulhearn** [comment]
Hey, sorry I missed the .gitignore conflict, should be resolved now. thanks for the feedback!

**Pull request merged**